integration-tests/maven/src/test/java/io/quarkus/maven/it/SetupIT.java

```diff
@@ -0,0 +1,94 @@
+package io.quarkus.maven.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.*;
```

Comment:
It should not use * imports

Reply:
I replaced all * imports with explicit ones